JapaneseFilter/sections/whitelist.txt

```diff
@@ -1,6 +1,11 @@
 !
 ! White list. Fixing filtration errors(false-positive)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/55827
+!+ PLATFORM(android)
+||yads.c.yimg.jp/js/yads-async.js$script,redirect=noopjs,domain=map.yahoo.co.jp
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
```

Comment:
If it is mobile only, why ext_safari then? Also if it is mobile only, please move the Mobile ads filter.

Reply:
Is it okay now?

Reply:
Looks OK